src/main/kotlin/app/revanced/patches/youtube/misc/videoid/patch/VideoIdPatch.kt

```diff
@@ -35,12 +35,14 @@ class VideoIdPatch : BytecodePatch(
 
         injectCall("Lapp/revanced/integrations/videoplayer/VideoInformation;->setCurrentVideoId(Ljava/lang/String;)V")
 
+        // offset so setCurrentVideoId is called before any injected call
+        offset++
+
         return PatchResultSuccess()
     }
 
     companion object {
-        // move-result-object offset
-        private const val offset = 2
```

Comment:
Want to add this comment back?

Reply:
No, this has been moved to the correct location somewhere below.